server/testdata/endpoints/18_couper.hcl

```diff
@@ -0,0 +1,32 @@
+server "couper" {
+  endpoint "/abcdef" {
+    proxy = "test"
+    response {
```

Comment:
The `proxy` attribute should be a replacement for a `proxy` block, which does not require a `response` block.

Reply:
In this case its the same as:

```
endpoint "/abcdef" {
  proxy "abcdef" {
    url = "${env.COUPER_TEST_BACKEND_ADDR}/anything"
  }
}
```

The `proxy = "defaultName"` below does not require a `response` block